package.json

```diff
@@ -37,7 +37,12 @@
     "@types/semver": "^5.5.0",
     "ansicolors": "^0.3.2",
     "debug": "^3.2.5",
-    "lodash": "^4.17.15",
+    "lodash.assign": "^4.2.0",
```

Comment:
you can remove `@types/lodash` above, as we're not importing `lodash` proper any more

Reply:
missed those :+1: